src/test/java/de/bonndan/nivio/appuser/AppUserRoleTest.java

```diff
@@ -0,0 +1,18 @@
+package de.bonndan.nivio.appuser;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
org.hamcrest is a different library than assertJ which we usually use.

Reply:
Changed it to junit